mods/d2k/fluent/rules.ftl

```diff
@@ -17,7 +17,7 @@ notification-base-under-attack = Base under attack.
 notification-ally-under-attack = Our ally is under attack.
 notification-harvester-under-attack = Harvester under attack.
 notification-silos-needed = Silos needed.
-notification-no-room-for-new-units = No room for new unit.
+notification-no-room-for-new-units = No room for new units.
```

Comment:
The voice lines clearly say "No room for new unit"
![image](https://github.com/user-attachments/assets/62a57e3e-ab0f-4b1c-a95b-4c996348e564)